tests/governance.py

```diff
@@ -20,13 +20,13 @@
 
 
 def run(args):
-    hosts = ["localhost", "localhost"]
+    hosts = ["localhost"] * (4 if args.consensus == "pbft" else 2)
```

Comment:
Could the number of hosts be automatically derived from `args.consensus` without having to specify it for each test?